test/utils/file.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'chai';
+const fs = require("fs");
+import { getSoftwareHashedValuesFile, getStoredHashedValues, storeHashedValues } from '../../src/utils/file';
+
+describe("file util", function () {
+  beforeEach(function () {
+    deleteHashedFile();
+  });
+
+  afterEach(function () {
+    deleteHashedFile();
+  });
+
```

Comment:
to make it easier to read the tests and output, each of the `it` blocks should have a context around them describing the setup
```
Context 'when no file exists' 
  it 'returns an empty hash'

context 'when a file exists'
  it 'returns a hash with the file value'

```

something like that

Reply:
cool, i'll update that, but both tests validate when the file doesn't exist. its supposed to show that the initial install won't throw an error when trying to read it for the first time. I'll update the contexts for that info and what you mentioned.